bot/start.js

```diff
@@ -480,6 +481,31 @@ const initialize = (botToken, options) => {
     }
   });
 
+  bot.command('setaddress', async (ctx) => {
+    try {
+      const user = await validateUser(ctx, bot, false);
+
+      if (!user)
+          return;
+
+      let [ lightningAddress ] = await validateParams(ctx, bot, user, 2, '<lightningAddress>');
```

Comment:
Recently I changed validateParams() to work with markdown, please change this line to this one
```
let [ lightningAddress ] = await validateParams(ctx, bot, user, 2, '\\<_lightningAddress_\\>');
```